spec/models/database_change_protection_spec.rb

```diff
@@ -90,6 +90,7 @@ def perform
     describe "#delete" do
       it "refuses if journaled columns exist" do
         expect { subject.delete }.to raise_error(/aborted by Journaled/)
+        expect { subject.destroy! }.not_to raise_error
```

Comment:
this errors:

```
Failures:

  1) Raw database change protection an instance #delete refuses if journaled columns exist
     Failure/Error: expect { subject.destroy! }.not_to raise_error

       expected no Exception, got #<RuntimeError: #delete_all aborted by Journaled::Changes.

       Call .destroy_all instead to ensure journaling or invoke .delete_all(force: true)
       to override and skip journaling.
       > with backtrace:
         # ./lib/journaled/relation_change_protection.rb:21:in `delete_all'
         # ./spec/models/database_change_protection_spec.rb:93:in `block (5 levels) in <top (required)>'
         # ./spec/models/database_change_protection_spec.rb:93:in `block (4 levels) in <top (required)>'
     # ./spec/models/database_change_protection_spec.rb:93:in `block (4 levels) in <top (required)>'
```

Reply:
nice I spose we can use stack analysis to go ahead if we're being called from within destroy? :-D

Reply:
lol sure :P